manifests/params.pp

```diff
@@ -5,7 +5,7 @@
 class jira::params {
   Exec { path => ['/bin/', '/sbin/', '/usr/bin/', '/usr/sbin/'] }
 
-  $service_file_location = '/usr/lib/systemd/system/jira.service'
+  $service_file_location = '/lib/systemd/system/jira.service'
```

Comment:
can't we put it into /etc/systemd/system?

Reply:
I wouldn't mind that, but it does mean a migration. This was the easiest fix to the regression I introduced.

Reply:
okay, fine by me as a short term fix